buidler.config.js

```diff
@@ -134,7 +134,8 @@ module.exports = {
             "0x28d1bfbbafe9d1d4f5a11c3c16ab6bf9084de48d99fbac4058bdfa3c80b2908f",
           balance: "100000000000000000000000000000000"
         }
-      ]
+      ],
+      allowUnlimitedContractSize: true
```

Comment:
why do we need this part?

Reply:
The malicious network contracts exceeds the max bytecode size. Without this, tests will fail cos the contracts can't be deployed (out of gas txns).